en-US/focus-ios.xliff

```diff
@@ -931,6 +930,26 @@
         <target>Site missing content or acting strange?</target>
         <note>Text for a label that indicates the title for sites not working tip.</note>
       </trans-unit>
+      <trans-unit id="TodayWidget.QuickActionGalleryDescription">
+        <source>Add a Firefox shortcut to your Home screen. After adding the widget, touch and hold to edit it and select a different shortcut.</source>
+        <target>Add a Firefox shortcut to your Home screen. After adding the widget, touch and hold to edit it and select a different shortcut.</target>
+        <note>Description for small size widget to add it to home screen</note>
+      </trans-unit>
+      <trans-unit id="TodayWidget.QuickActionsGalleryTitle">
+        <source>Quick Actions</source>
+        <target>Quick Actions</target>
+        <note>Quick Actions title when widget enters edit mode</note>
+      </trans-unit>
+      <trans-unit id="TodayWidget.SearchInApp">
+        <source>Search in</source>
```

Comment:
This assumes that all locales can leave the app name at the end, and it doesn't scale for80 languages.

This string should have a placeable for the app name, and accept that it might span more than 2 lines.

Reply:
Should be fixed by #3435